neuro_flow/expr.py

```diff
@@ -769,8 +894,8 @@ class SimpleOptStrExpr(StrExprMixin, Expr[str]):
 
 
 class IdExprMixin:
-    @classmethod
-    def convert(cls, arg: str) -> str:
+    def convert(self, arg: TypeT) -> str:
+        assert isinstance(arg, str)
```

Comment:
`_try_convert` only handles `TypeError` and `ValueError`. So, look like current code will produce strange error for the following yaml:
```
tasks:
  - id: ${{ ["22", "33"] }}
```
I think it is better to raise `TypeError` directly here.

Reply:
Done